lib/Doctrine/Query/Tokenizer.php

```diff
@@ -94,7 +94,7 @@ public function tokenizeQuery($query)
                 break;
 
                 case 'by':
-                    continue;
```

Comment:
I'm not sure why we didn't catch these when running against php 7.2...

They don't cause the tests to fail at all, but do spit out a warning at the beginning of the test run.

I changed to `break` instead of `continue 2` because `break` is how PHP would interpret these. In these cases, `continue 2` would have been fine as well as there is no executable code after the switch (that's still inside the foreach loop).